tests/test_api.py

```diff
@@ -301,6 +348,28 @@ async def test_pass_through(client, parameters, expected):
         assert request.pass_through is expected
 
 
+@pytest.mark.skipif(
+    os.environ.get("PASS_THROUGH") is None, reason="External pass-through disabled"
+)
+@pytest.mark.asyncio
+async def test_external_pass_through(client):  # pragma: nocover
```

Comment:
Added a test case for a real external pass-through call, for easier testing/verifying that #68 still works.

The test is skipped by default, to not call remote, but can be run individually with:
```bash
$ PASS_THROUGH=1 nox -s test -- -k test_external_pass_through
```

Pass-through responses still seams to work, but a second pair of eyes is appreciated @bhumilsarvaiya ;-)